### PR TITLE
Backport 52299 to 5.0

### PR DESCRIPTION
**Pull request by kopancek**
## Description

backport #52299 to 5.0

I needed to change the parent of the migration in metadata.yaml and there was a bazel conflict. I did it manually, not sure if there's a better process for it...

## Test plan

Tested locally that it still builds...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ All notable changes to Sourcegraph are documented in this file.
 - Fixed an issue where a Code Insights query with structural search type received 0 search results for the latest commit of any matching repo. [#51076](https://github.com/sourcegraph/sourcegraph/pull/51076)
 - GitLab code host connections will disable repo-centric repository permission syncs when the authentication provider is set as "oauth". This prevents repo-centric permission sync from getting incorrect data. [#51452](https://github.com/sourcegraph/sourcegraph/pull/51452)
 - Code intelligence background jobs did not correctly use an internal context, causing SCIP data to sometimes be prematurely deleted. [#51591](https://github.com/sourcegraph/sourcegraph/pull/51591)
+- Slow request logs now have the correct trace and span IDs attached if a trace is present on the request. [#51826](https://github.com/sourcegraph/sourcegraph/pull/51826)
```
Comment (BolajiOlajide) [inline CHANGELOG.md:52]: This seems unrelated to this backport PR.

**Review by BolajiOlajide — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kopancek) [reply CHANGELOG.md:52]: Agreed, probably a bad cherry-pick merge of the changelog. Removing...
